alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/GrpcServerConfig.java

```diff
@@ -55,6 +55,9 @@
   @Value("${alpha.server.ssl.clientCert:client.crt}")
   private String clientCert;
 
+  @Value("${alpha.feature.native:false}")
+  private boolean nettyTransport;
```

Comment:
How about Rename the configuration setting to nativetransport.

Reply:
done